### PR TITLE
Move syrups with minus time remaining to ended

### PR DESCRIPTION
**Pull request by totop716**

### CODE REVIEW

File: src/state/stake/hooks.ts [hunk 1]
```diff
@@ -194,14 +194,13 @@ export function useSyrupInfo(
...
-              tokenToFilterBy === undefined || tokenToFilterBy === null
-                ? getSearchFiltered(
-                    stakingRewardInfo,
-                    filter ? filter.search : '',
-                  )
-                : tokenToFilterBy.equals(stakingRewardInfo.token) &&
-                  tokenToFilterBy.equals(stakingRewardInfo.token),
+            .filter(
+              (syrupInfo) =>
+                syrupInfo.ending > Math.floor(Date.now() / 1000) &&
```
Comment (FriskyHamTitz) [inline src/state/stake/hooks.ts:199]: Make the / 1000 more clear e.g method should be like 
GetDateInSeconds etc.

I'm the future times should be named in their stored value e.g. EndingDateInSeconds

File: src/state/stake/hooks.ts [hunk 2]
```diff
@@ -194,14 +194,13 @@ export function useSyrupInfo(
...
+                (tokenToFilterBy === undefined || tokenToFilterBy === null
+                  ? getSearchFiltered(syrupInfo, filter ? filter.search : '')
+                  : tokenToFilterBy.equals(syrupInfo.token) &&
+                    tokenToFilterBy.equals(syrupInfo.token)),
```
Comment (FriskyHamTitz) [inline src/state/stake/hooks.ts:203]: What is this checking?
tokenToFilterBy.equals(syrupInfo.token) &&
                    tokenToFilterBy.equals(syrupInfo.token)

Looks like the same condition

File: src/state/stake/hooks.ts [hunk 3]
```diff
@@ -392,23 +391,23 @@ export function useOldSyrupInfo(
...
-                  tokenToFilterBy.equals(stakingRewardInfo.token),
-            ) ?? []
-        : [],
-    [chainId, tokenToFilterBy, startIndex, endIndex, filter],
-  );
+  const info = useMemo(() => {
+    if (!chainId) return [];
+    const endedSyrupInfos =
+      returnSyrupInfo(false)[chainId]?.filter(
+        (syrupInfo) => syrupInfo.ending <= Math.floor(Date.now() / 1000),
```
Comment (FriskyHamTitz) [inline src/state/stake/hooks.ts:398]: Same comments as above

File: src/state/stake/hooks.ts [hunk 4]
```diff
@@ -392,23 +391,23 @@ export function useOldSyrupInfo(
...
+      ) ?? [];
+    const oldSyrupInfos = returnSyrupInfo(true)[chainId] ?? [];
+    const allOldSyrupInfos = endedSyrupInfos.concat(oldSyrupInfos);
+    return allOldSyrupInfos
+      .slice(startIndex, endIndex)
+      .filter((syrupInfo) =>
+        tokenToFilterBy === undefined || tokenToFilterBy === null
+          ? getSearchFiltered(syrupInfo, filter ? filter.search : '')
```
Comment (FriskyHamTitz) [inline src/state/stake/hooks.ts:406]: Same comments as above

File: src/state/stake/hooks.ts [hunk 5]
```diff
@@ -539,7 +538,7 @@ export function useOldSyrupInfo(
           memo.push({
             stakingRewardAddress: rewardsAddress,
             token: syrupInfo.token,
-            ended: syrupInfo.ended,
```
Comment (FriskyHamTitz) [inline src/state/stake/hooks.ts:542]: Do we need that field in the JSON metadata anymore?

**Review by FriskyHamTitz — CHANGES_REQUESTED** [review]

File: src/state/stake/hooks.ts (hunk 5 above)
Comment (totop716) [reply src/state/stake/hooks.ts:542]: I think we do not need ended field for syrup

**Review by FriskyHamTitz — APPROVED** [review]

**Review by sameepsi — APPROVED** [review]

**Pull request merged**